### PR TITLE
Add info for enabling the workflow

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #34741  .

### Summary of Changes
Add some information how to enable the workflow in J4.

### Testing Instructions
Code review 
or
apply patch, install sample data in a fresh installation and check the workflow text.

### Expected result AFTER applying this Pull Request
grafik

### CODE REVIEW

**Comment by Valc** [comment]
:+1:

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 3e09bc0a3cf2b8e59475d2a43a4bafe4a99c98d7<br /><br />Language strings look good for this PR.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35536">issues.joomla.org/tracker/joomla-cms/35536</a>.</sub>

**Comment by jwaisner** [comment]
@Valc if you have tested, please mark your result on the tracker to confirm if successful.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35536">issues.joomla.org/tracker/joomla-cms/35536</a>.</sub>

**Comment by Valc** [comment]
I have tested this item :white_check_mark: successfully on 3e09bc0a3cf2b8e59475d2a43a4bafe4a99c98d7<br /><br />It is working fine here<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35536">issues.joomla.org/tracker/joomla-cms/35536</a>.</sub>

**Comment by Valc** [comment]
Hi!
Is there other opened issue tracker to test about add a similar message at "post installation system messages"?
Due as new "hidden" by default and cool feature, now i am not able to see that one notified without install the same data.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35536">issues.joomla.org/tracker/joomla-cms/35536</a>.</sub>

**Comment by richard67** [comment]
@chmst Wouldn't an ordered list haven been more suitable for the 3 steps for activating?

File: administrator/language/en-GB/plg_sampledata_blog.ini [hunk 1]
```diff
@@ -41,7 +41,7 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_8_INTROTEXT="<p><strong>We proudl
...
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_FULLTEXT=""
-PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Workflows manage the stages your articles must go through until they are published.<br>We have activated a workflow for the sample data blog.<br>In the administration process your articles through the different stages.</p>"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Workflows manage the stages your articles must go through until they are published.</p><p>The component to manage workflows is not enabled by default.</p><p>If you want to be able to see the workflow that we have added for sample data, you first need to activate the functionality</p><ul><li>Access the administration area</li><li>From 'Content' > 'Articles' > 'Options' > 'Integration' tab set 'Enable workflow' to 'Yes'</li><li>Save changes</li></ul><p>From here when you access 'Content' again you should be able to see the section 'Workflows'</p><p>Also, when you edit an article you should be able to see the new transitions for articles related to workflows.</p>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_sampledata_blog.ini]:
```suggestion
PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Workflows manage the stages your articles must go through until they are published.</p><p>The component to manage workflows is not enabled by default.</p><p>If you want to be able to see the workflow that we have added for sample data, you first need to activate the functionality.</p><ul><li>Access the administration area</li><li>From 'Content' > 'Articles' > 'Options' > 'Integration' tab set 'Enable workflow' to 'Yes'</li><li>Save changes</li></ul><p>From here when you access 'Content' again you should be able to see the section 'Workflows'.</p><p>Also, when you edit an article you should be able to see the new transitions for articles related to workflows.</p>"
```

Comment (chmst) [reply administrator/language/en-GB/plg_sampledata_blog.ini]: thank you, done.

**Comment by chmst** [comment]
Nice idea, @richard67. Please test.

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on c9a4b9d30b91965aaad5cfe8d6f53ddb10c55082<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35536">issues.joomla.org/tracker/joomla-cms/35536</a>.</sub>

**Comment by ChristineWk** [comment]
Result:

screen shot 2021-09-19 at 21 30 49<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35536">issues.joomla.org/tracker/joomla-cms/35536</a>.</sub>

File: administrator/language/en-GB/plg_sampledata_blog.ini [hunk 2]
```diff
@@ -41,7 +41,7 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_8_INTROTEXT="<p><strong>We proudl
...
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_FULLTEXT=""
-PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Workflows manage the stages your articles must go through until they are published.<br>We have activated a workflow for the sample data blog.<br>In the administration process your articles through the different stages.</p>"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Workflows manage the stages your articles must go through until they are published.</p><p>The component to manage workflows is not enabled by default.</p><p>If you want to be able to see the workflow that we have added for sample data, you first need to activate the functionality.</p><ol><li>Access the administration area</li><li>From 'Content' > 'Articles' > 'Options' > 'Integration' tab set 'Enable workflow' to 'Yes'</li><li>Save changes</li></ol><p>From here when you access 'Content' again you should be able to see the section 'Workflows'.</p><p>Also, when you edit an article you should be able to see the new transitions for articles related to workflows.</p>"
```
Comment (brianteeman) [inline administrator/language/en-GB/plg_sampledata_blog.ini]:
```suggestion
PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Workflows manage the stages your articles must go through until they are published.</p><p>The component to manage workflows is not enabled by default.</p><p>To see the workflow that we have provided with the sample data, you first need to enable this functionality.</p><ol><li>Access the administration area</li><li>From 'Content' > 'Articles' > 'Options' > 'Integration' tab set 'Enable workflow' to 'Yes'</li><li>Save changes</li></ol><p>Now when you access 'Content' again you will see the section 'Workflows'.</p><p>When you edit an article you will also see the new transitions for articles related to workflows.</p>"
```

**Comment by ditsuke** [comment]
I have tested this item :white_check_mark: successfully on c9a4b9d30b91965aaad5cfe8d6f53ddb10c55082<br /><br />Looks good to me.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35536">issues.joomla.org/tracker/joomla-cms/35536</a>.</sub>

**Comment by richard67** [comment]
@chmst Could you apply @brianteeman 's suggestions above? Thanks in advance.

**Comment by chmst** [comment]
thanks @brianteeman

**Comment by richard67** [comment]
I've restored the previous two test results since the only change after that was a change in wording.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35536">issues.joomla.org/tracker/joomla-cms/35536</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**